src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -154,15 +175,8 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig, val beamServices: BeamSe
           case Some(skim) =>
             skim
           case None =>
-            SkimInternal(
-              time = 1.1,
-              generalizedTime = 1.1,
-              beamServices.beamConfig.beam.agentsim.agents.rideHail.pooledToRegularRideCostRatio * 1.1,
-              distance = 0,
-              beamServices.beamConfig.beam.agentsim.agents.rideHail.pooledToRegularRideCostRatio,
-              count = 0,
-              energy = 1.1
-            )
+            val cost = getRideHailCost(RIDE_HAIL_POOLED, solo.distance, solo.time)
+            return (1.1, cost / solo.cost)
```

Comment:
`return`s are dangerous in scala - Can you rework this to not use it? Ping me if you'd like to discuss ideas on how to do it.

Reply:
Thanks for these!